src/components/people/PersonPicker/index.tsx

```diff
@@ -25,6 +25,12 @@ type PersonPickerProps = {
     hasError?: boolean;
     errorMessage?: string;
     onSelect?: (person: PersonDetails) => void;
+    customSectionGenerator?: (
```

Comment:
make a type or interface for PersonPickerSectionFn
in props: {sectionFn = peopleToSection}
in effect only call sectionFn